### PR TITLE
refactor: coffee -> es6

### PR DESCRIPTION
**Pull request by loddit**
去除 #3 里面的 coffee

### CODE REVIEW

File: legilimens.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+const getLastedReleaseTime = new Promise((resolve, reject) => {
+  callGithubAPI({
+    url: repoUrl + RELEASES_PATH,
+    callback(error, response, body) {
+      switch (response.statusCode) {
+        case 200:
+          const createdAt = ((JSON.parse(body).filter(release => {
+            return ((release.target_commitish === 'master') || isHotfix) && !release.prerelease;
+          }))[0], x => x.created_at);
```
Comment (chafel) [inline legilimens.js]: `const createsAt = (XXX, x => x.created_at)` 看不懂

File: legilimens.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+function getClosedPullRequestsAfter(time) {
+  callGithubAPI({
+    url: repoUrl + RECENT_CLOSED_PR_PATH,
+    callback(error, response, body) {
+      if (!error && (response.statusCode !== 200)) {
+        return console.log(error, body);
+      } else {
+        const pullRequests = JSON.parse(body)
+         .filter(pullRequest => new Date(pullRequest.merged_at) > time).filter(pullRequest => pullRequest.base.ref === baseBranch);
```
Comment (chafel) [inline legilimens.js:60]: 用这个吧 https://help.github.com/articles/searching-issues-and-pull-requests/#search-based-on-when-a-pull-request-was-merged
就不用拉下来一堆再 filter 了

Comment (loddit) [reply legilimens.js:60]: 你给的是 web 界面的功能，API 没有这个参数
https://developer.github.com/v3/pulls/#list-pull-requests

**Comment by chafel** [comment]
LGTM

**Pull request merged**